### ISSUE TITLE
Add aria-label to zoom to extent?

### ISSUE THREAD

**Issue by chughes-lincoln** [geomoose/gm3#973]
I'm working on ironing out some of the things that aren't wcag compliant, and the jump-to-extent select box was flagged by some of the checkers I was using as it doesn't have a label. It does have a placeholder value that is read by one of the screen readers I was testing it with, but my understanding is that the placeholder on its own isn't wcag compliant as it isn't consistently read by screen readers.

I found a couple ways of fixing this that pass the tests. The first is just wrapping the jump-to-extent span in the index.html with a label element (which I would rather avoid as it adds redundant text in the footer). The second is adding an aria-label in the select element in the jumpToExtent.js file before rebuilding the javascript.

See the code change below:
```
return (
    <select
      aria-label={t("zoomto-extent")}
      value="default"
      onChange={(evt) => {
        onZoomTo(locations[evt.target.value].extent);
      }}
    >
      <option disabled key="default" value="default">
        {t("zoomto-extent")}
      </option>
      {options}
    </select>
  );
};

```
